doc/lua_api.txt

```diff
@@ -1477,6 +1477,9 @@ Currently supported flags: `place_center_x`, `place_center_y`, `place_center_z`,
 * `force_placement`: Schematic nodes other than "ignore" will replace existing
   nodes.
 
+The X and Z flags refer to centering before any rotation around the Y axis is
+applied.
```

Comment:
Surely this is unnecessary information since the point of the PR is fixing the issue this caused?

Reply:
Actually it still applies even with the fix and is not the cause of the bug. It's useful information for how to use the centering flags when rotation is involved, and is closely related to the PR.
For example a log schematic that is defined in a lua table as being long in X but is rotated into the Z axis should use the 'place center x' flag.

Reply:
But that isn't what this says. This instead reads like a warning that it can't account for the rotation.

Reply:
Hmm i don't read it that way, however it is not clear, will improve it.

Reply:
Nah i'll leave docs out of this PR.

Reply:
Removed.